### ISSUE TITLE
TOR Tracking Eye of Sauron

### ISSUE THREAD

**Issue by fattom23** [BernhardPosselt/foundryvtt-special-dice-roller#27]
Under certain circumstances (such as when a character is miserable), whether or not an Eye of Sauron is rolled can be important. Is that information available when accessing the rolls programatically? I'm trying to integrate the special rolls into my TOR system, and it would be really helpful to be able to track if the f die was a 1, so that I can provide a notification that a Bout of Madness should be implemented.

**Comment by BernhardPosselt** [comment]
Yep, you want to basically use the code in rollFormula on your own: https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/blob/master/src/roller.ts#L67-L76

The roll method will give you an array of Dice and Faces see https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/blob/master/src/roller.ts#L7

Those are basically just numbers from 0..n (typical typescript enums) that you can look up here https://github.com/BernhardPosselt/foundryvtt-special-dice-roller/blob/master/src/tor/dice.ts#L4-L32

Access to the roller is available on the game object as described here https://github.com/BernhardPosselt/foundryvtt-special-dice-roller#rolling-dice-programmatically
